extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -5,497 +5,536 @@
+		['set', 'dimension3', window.cscoreCat],                       // Hub
+		['set', 'dimension4', window.skin],                            // Skin
+		['set', 'dimension5', !!window.wgUserName ? 'user' : 'anon']  // LoginStatus
+	);
+
+	/*
+	 * Remove when SOC-217 ABTest is finished
+	 */
+	/**
+	 * Get unconfirmed email AbTest user type
+	 * @returns {string}
+	 */
+	function getUnconfirmedEmailUserType() {
+		if (!window.wgUserName) {
+			return 'anon';
+		} else {
+			switch (window.wgNotConfirmedEmail) {
+				case '1':
+					return 'unconfirmed';
+				case '2':
+					return 'confirmed';
+				default:
+					return 'old user';
+			}
+		}
+	}
+	/*
+	 * end remove
+	 */
+
+	/**** Medium-Priority Custom Dimensions ****/
+	_gaWikiaPush(
+		['set', 'dimension8', window.wikiaPageType],                            // PageType
+		['set', 'dimension9', window.wgCityId],                                 // CityId
+		['set', 'dimension14', window.wgGaHasAds ? 'Yes' : 'No'],               // HasAds
+		['set', 'dimension15', window.wikiaPageIsCorporate ? 'Yes' : 'No'],     // IsCorporatePage
+		['set', 'dimension16', getKruxSegment()],                               // Krux Segment
+		['set', 'dimension17', window.wgWikiVertical],                          // Vertical
+		['set', 'dimension18', window.wgWikiCategories.join(',')],              // Categories
+		['set', 'dimension19', window.wgArticleType],                           // ArticleType
+		['set', 'dimension20', window.wgABPerformanceTest || 'not set'],        // Performance A/B testing
+		['set', 'dimension21', String(window.wgArticleId)],                     // ArticleId
+		['set', 'dimension23', window.wikiaIsPowerUserFrequent ? 'Yes' : 'No'], // IsPowerUser: Frequent
+		['set', 'dimension24', window.wikiaIsPowerUserLifetime ? 'Yes' : 'No'], // IsPowerUser: Lifetime
+		['set', 'dimension25', String(window.wgNamespaceNumber)]                // Namespace Number
+	);
+
+	/*
+	 * Remove when SOC-217 ABTest is finished
+	 */
+	_gaWikiaPush(['set', 'dimension39', getUnconfirmedEmailUserType()]);      // UnconfirmedEmailUserType
+	/*
+	 * end remove
+	 */
+
+	/**
+	 * Checks if Optimizely object and its crucial data attributes are available
+	 *
+	 * @returns {boolean}
+	 */
+	function isOptimizelyLoadedAndActive() {
+		var optimizely = window.optimizely;
+
+		return optimizely &&
+			optimizely.activeExperiments &&
+			Array.isArray(optimizely.activeExperiments) &&
+			optimizely.activeExperiments.length > 0 &&
+			typeof optimizely.allExperiments === 'object' &&
+			Object.keys(optimizely.allExperiments).length > 0 &&
+			typeof optimizely.variationNamesMap === 'object' &&
+			Object.keys(optimizely.variationNamesMap).length > 0;
+	}
+
+	// UA integration code is also used in Mercury SPA - if you change it here, change it there too:
+	// https://github.com/Wikia/mercury/blob/XW-6/front/scripts/mercury/utils/variantTesting.ts#L75
```

Comment:
This link won't work after deleting the branch. Would be safer too give a link to file in dev branch and a name of the function.